src/main/java/eu/dissco/backend/service/SpecimenService.java

```diff
@@ -287,18 +287,30 @@ private Map<String, List<String>> mapParams(MultiValueMap<String, String> params
 
   public JsonApiWrapper aggregations(MultiValueMap<String, String> params, String path)
       throws IOException, UnknownParameterException {
-    var aggregations = elasticRepository.getAggregations(mapParams(params));
-    var dataNode = new JsonApiData(String.valueOf(params.hashCode()), "aggregations",
+    var aggregations = elasticRepository.getAggregations(mapParamsKeyword(params));
+    var dataNode = new JsonApiData(String.valueOf(params.hashCode()), AGGREGATIONS_TYPE,
         mapper.valueToTree(aggregations));
     return new JsonApiWrapper(dataNode, new JsonApiLinks(path));
   }
 
   public JsonApiWrapper discipline(String path) throws IOException {
     var disciplineResult = elasticRepository.getAggregation(TOPIC_DISCIPLINE);
-    var dataNode = new JsonApiData(String.valueOf(path.hashCode()), "aggregations",
+    var dataNode = new JsonApiData(String.valueOf(path.hashCode()), AGGREGATIONS_TYPE,
         mapper.valueToTree(disciplineResult.getRight()));
     return new JsonApiWrapper(dataNode,
         new JsonApiLinks(path), new JsonApiMeta(disciplineResult.getLeft()));
   }
 
+  public JsonApiWrapper searchTermValue(String name, String value, String path)
```

Comment:
"name" here means term name? In the controller this variable is called "term". That's more clear to me, but I see that "name" is an attribute of Term, so i see why it's called this.

Reply:
Yes tricky with all the different names. For the frontend it is the term, but for elastic it is the name... So frontend would like to keep it term. But here it is not really the full term but the (simplified) name of the term. Does that make sense?